Sharpmake/Util.cs

```diff
@@ -145,20 +145,27 @@ public static string GetTextTemplateDirectiveParam(string filePath, string direc
         /// <returns>true=equal, false=not equal</returns>
         private static bool AreStreamsEqual(Stream stream1, Stream stream2)
         {
+#if NET5_0_OR_GREATER
```

Comment:
Why are you adding these ifdef? We killed support for .net version older than 5.0 a few months ago.

Reply:
Oh that's my bad, I forgot to remove those after updating Sharpmake. They should be gone now.